tools/src/main/python/dlpx/virtualization/_internal/codegen/templates/base_model_.mustache

```diff
@@ -276,14 +282,15 @@ class GeneratedClassesTypeError(GeneratedClassesError):
         if not required and parameter is None:
             return None
         # Now check if the types are incorrect.
+        num_types = (float, complex, int)
```

Comment:
did you purposely leave out `long`? is that not a type in py3 anymore?

Reply:
Correct, `long` is no longer a type in py3. For numeric types it's just the 3 we have here now.